api/src/infrastructure/docker.rs

```diff
@@ -399,14 +434,14 @@ impl DockerInfrastructure {
                 Err(err) => debug!("Could not clean up image: {:?}", err),
             }
         }
-
         Ok(Service::try_from(&container_details)?)
     }
 
     fn create_container_options(
         app_name: &String,
         service_config: &ServiceConfig,
         container_config: &ContainerConfig,
+        host_config_binds: &Vec<String>,
```

Comment:
`cargo clippy` should recommend to use following syntax (from the top of my head):

```suggestion
        host_config_binds: &[&str],
```

This change make it more usable in more context. 

In the past we that wrong too if you look at the `app_name` parameter. As you are changing the signature, you could fix that for this function too.